pkg/controller/config/v1alpha1/types.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ExtensionConfig is the configuration for the os-coreos extension.
+type ExtensionConfig struct {
+	metav1.TypeMeta `json:",inline"`
+	// UseNTP whether to toggle NTP or not
+	UseNTP *bool `json:"useNTP"`
```

Comment:
Could this be moved into the NTPConfig struct? Maybe something like `enabled` as field name

Reply:
It could I guess. I did however use the entire `ntpConfig` struct as k,v for templating.

I don't have a strong opinion about this, we can do either way.

Reply:
Do we even need this bool field at all? If there are NTP servers listed in the configuration they should be used right?

Reply:
I guess that would be implied then, yes